identity-vault/getting-started/src/util/session-vault.ts

```diff
@@ -71,25 +68,28 @@ export const clearSession = async (): Promise<void> => {
 };
 
 export const updateUnlockMode = async (mode: UnlockMode): Promise<void> => {
-  const type =
-    mode === 'BiometricsWithPasscode'
-      ? VaultType.DeviceSecurity
-      : mode === 'InMemory'
-      ? VaultType.InMemory
-      : VaultType.SecureStorage;
-  const deviceSecurityType =
-    type === VaultType.DeviceSecurity
-      ? DeviceSecurityType.Both
-      : DeviceSecurityType.None;
-  await vault.updateConfig({
-    ...(vault.config as IdentityVaultConfig),
-    type,
-    deviceSecurityType,
-  });
+  const newConfig = { ...(vault.config as IdentityVaultConfig) };
+
+  switch (mode) {
```

Comment:
This is missing the `deviceSecurityType` setting, from the previous version of this code. Also I think we should revert this change anyways to keep it consistent with several other repos. Sorry for the confusion.

Reply:
Shoot, thanks for catching